war/src/main/js/widgets/add/addform.less

```diff
@@ -281,14 +279,14 @@
   position: relative;
 }
 
-.j-item-options {
-
-  li:hover, li:focus {
+#createItem {
+  .add-item-copy:hover, .add-item-copy:focus,
+  .categories li:hover, li:focus {
```

Comment:
Don't make sense to consider the *option* copy at the same level than an item. Unrelated with the problem described in [JENKINS-33822](https://issues.jenkins-ci.org/browse/JENKINS-33822)